spec/idempotency/rails_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+class ApplicationController
```

Comment:
Could we also move this class to inside `describe` block? It's the same reason with moving `Response`.